pages/cloud/lake-effect.md

```diff
@@ -1,5 +1,69 @@
-# LakeEffect Subscriptions
+## LakeEffect Research Cloud _(On-Premise Cloud)_
+
+
+![](LakeEffectLogo_2.jpg)
+
+CCR hosts a subscription based on-premise cloud service affectionately known as **"Lake Effect"**, which provides access to virtual servers, storage and networking that users setup and manage themselves. It is what is referred to as Infrastructure as a Service and is equivalent in functionality to Amazon's EC2 Cloud service.
+
+Although high performance computing (HPC) is our largest and most widely used service, CCR recognizes that not all research requires number crunching on large scale computers (clusters). Lake Effect, was put into service to provide flexible infrastructure to support research computing such as webservers, databases, and long-running services that are not suited for HPC clusters. This along with our HPC platform allows us to provide a complete set of tools to support all the different research groups and collaborations througout UB.
+
+
+#### Lakeeffect Quick Start
+
+1. Details on Pricing and Subscription packages is [here](subscriptions.md)
+2. Request access through the [coldfront](../portals/coldfront.md) allocations management portal
+3. Once you have an active allocation, you can access the cloud through the [Dashboard](https://dashboard.cloud.ccr.buffalo.edu)
+4. Documenation on using Lakeeffect Cloud and Openstack:
+	- [Using LakeEffect Cloud](./using-lakeeffect.md)
```

Comment:
Could these be just different sections of this page instead of creating 2 new pages? We're trying to keep the number pages to a minimum. But if you think they warrant their own page, you have to add them to the nav section of `mkdocs.yml`. See [here](https://github.com/ubccr/ccrdocs/blob/main/mkdocs.yml#L40) for examples. 

If you create a page without adding it to the nav you'll see this warning: 
```
INFO     -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
              - cloud/subscriptions.md
              - cloud/using-lakeeffect.md
```